assets/linkage/dibbs_basic_algorithm.json

```diff
@@ -2,8 +2,8 @@
   "algorithm": [
     {
       "funcs": {
-        "first_name": "feature_match_fuzzy_string",
-        "last_name": "feature_match_exact"
+        "first_name": "func:recordlinker.linkage.matchers.feature_match_fuzzy_string",
```

Comment:
This is out of scope for this PR, but I think renaming the `feature_match_*` and `eval_*` functions down the line would be helpful. The current naming of the functions makes it so it's not entirely clear what they do.

For example, function names that communicate the action/command they are doing (e.g., `fuzzy_match`, `exact_match`) can be clearer I think, rather than those that describe how they are categorized in implementation.

Happy to help out with this in future refactoring efforts!

Reply:
Yeah 100%, I'd also like to change their signatures to accept data objects rather than lists of strings.  Its difficult to understand whats being compared when you have to memorize the order that PII shows up in a list.  Do you want to start a GH issue for this and put it in the backlog?